modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyQuery.jsx

```diff
@@ -266,9 +248,39 @@ function VocabularyQuery(props) {
           </MenuItem>
           );
       });
-      showUserEntry && suggestions.push(<Divider key="divider"/>);
     }
+
+    var allRequestsFailed = Object.keys(statuses).filter(vocab => !statuses[vocab]).length == 0;
+    var allRequestsSucceded = Object.keys(statuses).filter(vocab => statuses[vocab]).length == 0;
+
+    if (!allRequestsSucceded) {
+      if (suggestions.length > 0 && !allRequestsFailed) {
```

Comment:
`&& !allRequestsFailed` is unnecessary. At this point, if `suggestions.length > 0`, then  `!allRequestsFailed` is the only explanation. Anyway, we put a divider to visually separate from the items that already are in the `suggestions` and it doesn't matter how they got there.
```suggestion
      if (suggestions.length > 0) {
```

Reply:
Done in cb2b5e9